pywb/apps/frontendapp.py

```diff
@@ -559,6 +564,14 @@ def init_proxy(self, config):
         else:
             self.proxy_prefix = '/{0}/id_/'.format(proxy_coll)
 
+        self.proxy_timestamp = proxy_config.get('timestamp')
+        if self.proxy_timestamp:
+            if not self.ALL_DIGITS.match(self.proxy_timestamp):
+                try:
+                    self.proxy_timestamp = iso_date_to_timestamp(self.proxy_timestamp)
+                except:
```

Comment:
bare except: we should use `except Exception`

Reply:
ok, will update